test/e2e/data/shared/v1beta1_provider/metadata.yaml

```diff
@@ -7,12 +7,12 @@
 # one capz version at a time against one API version
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 releaseSeries:
-  - major: 1
```

Comment:
I think we need to keep this since we're still testing upgrade with it

Reply:
Shall I add back 1.11 or update the base version we upgrade from in [here](https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/4473/files#diff-35f348a5d567ebeba39b1e40bbab0e1a7f64009cbadff142952f9b9691e94666L86)?

Reply:
From ad-hoc discussion : will add back 1.11 to `test/e2e/data/shared/v1beta1_provider/metadata.yaml`

Reply:
Added back release 1.11.